packages/nextjs/app/builders/0xC52E3E454a735d4A74F3dd6Ea281F7AebDFAad83/page.tsx

```diff
@@ -0,0 +1,98 @@
+import Image from "next/image";
+import ProfilePic from "./pfp.png";
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const PersonalPage: NextPage = () => {
```

Comment:
You might want to give a more descriptive name to the page, since there are more builder pages.